### PR TITLE
Improve logs endpoint

### PR DESCRIPTION
**Pull request by ignaciojonas**

### CODE REVIEW

File: lib/auth0/api/v2/logs.rb [hunk 1]
```diff
@@ -1,9 +1,25 @@
...
+        # @param per_page [integer] The amount of entries per page. Default: 50. Max value: 100.
+        # @param sort [string] The field to use for sorting. 1 == ascending and -1 == descending.
+        # @param fields [string] A comma separated list of fields to include or exclude from the result.
+        # @param include_fields [boolean] True if the fields specified are to be included in the result, false otherwise.
+        # @param include_totals [string] True if a query summary must be included in the result, false otherwise.
+        # @param from [string] Log Event Id to start retrieving logs. You can limit the amount of logs using the take
+        # parameter.
+        # @param take [integer] The total amount of entries to retrieve when using the from parameter.
+        # Default: 50. Max value: 100.
+        #
```
Comment (falvarisqueta) [inline lib/auth0/api/v2/logs.rb:22]: should there be test for max values restriction of per_page  and take?

File: lib/auth0/api/v2/logs.rb [hunk 2]
```diff
@@ -16,16 +32,29 @@ def logs(options = {})
             from: options.fetch(:from, nil),
             take: options.fetch(:take, nil)
           }
-
-          path = '/api/v2/logs'
-          get(path, request_params)
+          if request_params[:take].to_i > 100
+            fail Auth0::MissingParameter, 'The total amount of entries should be less than 100'
```
Comment (leosoubeste) [inline lib/auth0/api/v2/logs.rb]: Would `Auth0::InvalidParameter` exception be more accurate?

File: spec/lib/auth0/api/v2/logs_spec.rb [hunk 3]
```diff
@@ -22,13 +23,19 @@
         take: nil)
       expect { @instance.logs }.not_to raise_error
     end
+    it 'is expect to rise an error when take is higher than 100' do
+      expect { @instance.logs(take: 200) }.to raise_error(
+        'The total amount of entries should be less than 100')
+    end
   end
```
Comment (falvarisqueta) [inline spec/lib/auth0/api/v2/logs_spec.rb:34]: sohuld we have same for per_page?

**Comment by leosoubeste** [comment]
looks good to me

**Pull request merged**